src/less/details/ds4/details.less

```diff
@@ -0,0 +1,38 @@
+@import "../../less/ds4/variables.less";
+
+details.details:not([open]) {
+    .details__summary--showless {
+        display: none;
+    }
+}
+
+details.details[open],
+details.details[data-makeup-open] {
+    .expand-btn__icon {
```

Comment:
@ianmcburnie should this be renamed since it's not a button?

Reply:
I think it's okay to stay the same. So we can recognize it as another nested module (expand button).

Reply:
If expansions only has one variation of button (i.e. no primary, secondary, etc). We can potentially get rid of the nested module and just have the CSS all contained within the details module.